src/build123d/operations_sketch.py

```diff
@@ -306,4 +305,9 @@ def trace(
         context.pending_edges = ShapeList()
 
     combined_faces = Face.fuse(*new_faces) if len(new_faces) > 1 else new_faces[0]
```

Comment:
```suggestion
    combined_faces = Face.fuse(*new_faces)
```
the `if len(to_fuse) < 2: return ShapeList(to_fuse)` special case test should go inside the `fuse()` method